sdk/python/symbolchain/Network.py

```diff
@@ -28,6 +29,16 @@ def public_key_to_address(self, public_key):
 
 		return self.create_address(version, checksum)
 
+	def is_valid_address_string(self, address_string):
+		"""Checks if an address string is valid and belongs to this network."""
+		if self.address_class.ENCODED_SIZE != len(address_string):
+			return False
+
+		if any(ch not in 'ABCDEFGHIJKLMNOPQRSTUVWXYZ234567' for ch in address_string):
```

Comment:
would probably throw it into a 'const' and name it BASE32_RFC4648_ALPHABET (just for the reference, that it's not custom one)